CRM/Core/DAO.php

```diff
@@ -2859,6 +2869,8 @@ public static function acceptedSQLOperators() {
       'NOT BETWEEN',
       'IS NOT NULL',
       'IS NULL',
+      'IS EMPTY',
+      'IS NOT EMPTY',
```

Comment:
@eileenmcnaughton I don't think it's necessary or advisable to add these to the list in CRM_Core_DAO. This list is shared between v3 api and the BAO query object and who knows what else.

APIv4 maintains its own list, which you can add to directly. Note that for example APIv4 accepts a "CONTAINS" operator which is not in this list either.